libs/designer/src/lib/core/parsers/BJSWorkflow/BJSDeserializer.ts

```diff
@@ -323,3 +333,46 @@ const throwIfMultipleTriggers = (definition: LogicAppsV2.WorkflowDefinition) =>
     );
   }
 };
+
+const addTriggerInstanceMetaData = (runInstance: LogicAppsV2.RunInstanceDefinition | null) => {
+  if (!isNullOrUndefined(runInstance)) {
```

Comment:
Just a tiny nit but something to help keep code cleaner, I do this all the time:
This is a condition where we can just invert the condition at the top, and return right away instead of having the whole function inside an if statement and then returning at the end